README.md

```diff
@@ -3,29 +3,34 @@
 </a-->
 
 # Codewind for VS Code
+Create and develop cloud-native, containerized web applications from VS Code.
 
 [![Marketplace](https://img.shields.io/vscode-marketplace/v/IBM.codewind.svg?label=marketplace&logo=visual-studio-code)](https://marketplace.visualstudio.com/items?itemName=IBM.codewind)
 [![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
 [![Slack](https://img.shields.io/badge/ibm--cloud--tech-blue.svg?logo=slack&label=slack)](https://slack-invite-ibm-cloud-tech.mybluemix.net/)
 
-- **[Documentation](https://microclimate.dev/codewindtechpreview)**
-- **[Changelog](https://github.com/microclimate-dev2ops/codewind-vscode/blob/master/CHANGELOG.md)**
-
-Create and develop cloud-native, containerized web applications from VS Code.
-
-## Getting started
+## Installing Codewind
+Prerequisites
+- Install [VS Code Version 1.27 or later](https://code.visualstudio.com/download).
 
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+<Add gif file here. The gif will begin with a title card and then cover all of the actions in the following section.>
+Complete both parts of the Codewind installation.
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or search for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+2. From the Codewind page in the VS Code Extensions Marketplace, click the **Install** button. The first part of Codewind installs.
+3. Go to the **Explorer** view group and open the **Codewind** view. The Codewind workspace is now added to the Explorer menu.
+4. To complete the second part of the installation, expand the Codewind workspace and click any of the functions. A notification window appears and asks if you would like to install the remaining Codewind component.
```

Comment:
> A notification window appears and asks if you would like to install the remaining Codewind component.
I don't think we have to say this. The user will see it for themselves.

Reply:
I also prefer 

`To complete the second part of the installation,`
->
`To complete the installation`

Reply:
I removed mention of "the second part," but I was under the impression that having two buttons to click for the installation was unusual and that it was good to ensure the user that the pop-up wasn't a result of them doing something wrong.

It's also not too out of the ordinary to write down the result of an action as part of a step. :)